### DISCUSSION TITLE
[Weekly Review Report]: May 27 - 31

### DISCUSSION THREAD

**Discussion by tabbassidaloii** [wikipathways/wikipathways-help#121] [curation-reports]
### Recent changes

About 26 pathways, most changes by [Eweitz](https://classic.wikipathways.org/index.php/User:Eweitz) and [Khanspers](https://classic.wikipathways.org/index.php/User:Khanspers).

### Under construction

for most of the pathways, the description and ontology tags should be added.
Added a quality tag where needed

### New pathways

* [WP5476](https://classic.wikipathways.org/index.php/Pathway:WP5476) by [Grisel](https://classic.wikipathways.org/index.php/User:Grisel)
[WP5475](https://classic.wikipathways.org/index.php/Pathway:WP5475) by [PhilipErnst](https://classic.wikipathways.org/index.php/User:PhilipErnst)
[WP5474](https://classic.wikipathways.org/index.php/Pathway:WP5474) by [Nberk](https://classic.wikipathways.org/index.php/User:Nberk)

### Deleted Pathways

No pathway

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

### Additional notes

[WP3931](https://classic.wikipathways.org/index.php/Pathway:WP3931) has one unannotated node "actr2b", though human pathway the gene symbol is in lowercase.

**Comment by khanspers** [comment]
Thank you @tabbassidaloii. I updated the unannotated node in WP3931.
